tests/unit/cmorizers/test_utilities.py

```diff
@@ -329,3 +330,17 @@ def test_read_cmor_config():
     assert 'thetao' in cfg['variables']
     assert 'Omon' in cfg['cmor_table'].tables
     assert 'thetao' in cfg['cmor_table'].tables['Omon']
+
+
+def test_make_time_suffix():
+    """Test the making of time suffix."""
+    fct = utils.make_time_suffix
+    # year / month / day / hour / minute
+    tp0 = DatetimeGregorian(2020, 1, 1, 10, 55)
+    tpn = DatetimeGregorian(2023, 4, 6, 20, 15)
+
+    assert fct("yr", tp0, tp0) == "202001-202012"
```

Comment:
Yearly data does not contain month information, can you check that in `make_time_suffix`?

Reply:
Updated, thanks for the heads-up.
Doing so I had to remove a condition from the original code base that was checking for cubes of len equal to 1 but I do not think that this piece of code was necessary to begin with.

Reply:
Test case added, it passes without issue.